judges/quality-of-service/quality-of-service.rb

```diff
@@ -95,4 +95,20 @@
     )[:total_count]
   end
   f.average_pull_rejection_rate = pulls.zero? ? 0 : rejected.to_f / pulls
+
+  # Average HOC and number of files changed in recent merged PRs
+  hocs = []
+  files = []
+  Fbe.unmask_repos.each do |repo|
+    Fbe.octo.search_issues(
+      "repo:#{repo} type:pr is:merged closed:>#{f.since.utc.iso8601[0..9]}"
+    )[:items].each do |json|
+      Fbe.octo.pull_request(repo, json[:number]).then do |pull|
+        hocs << (pull[:additions] + pull[:deletions])
+        files << pull[:changed_files]
+      end
+    end
+  end
+  f.average_pull_hoc_size = hocs.empty? ? 0 : hocs.inject(&:+).to_f / hocs.size
+  f.average_pull_files_size = files.empty? ? 0 : files.inject(&:+).to_f / files.size
```

Comment:
@Yegorov can we write it in a simpler way?
like this:
```suggestion
  f.average_pull_hoc_size = hocs.sum / hocs.size.to_f
  f.average_pull_files_size = files.sum / files.size.to_f
```

Reply:
@Suban05 I think not, because if hocs is empty, we will get an answer `NaN`
```ruby
3.3.4 :002 > 0 / 0.to_f
 => NaN
```
The same thing for `files`

Reply:
@Yegorov yes, indeed, you're right. what do you think about replacing `hocs.inject(&:+)` with `hocs.sum`?

Reply:
Yes, it has become more readable, thanks!